staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -63,9 +63,9 @@ type EtcdOptions struct {
 	WatchCacheSizes []string
 
 	// complete guards fields that must be initialized via Complete before the Apply methods can be used.
```

Comment:
shouldn't this commit be dropped if this is based on the latest version of https://github.com/kubernetes/kubernetes/pull/113529?

Reply:
I think we still need the `complete` bool to make sure we call the `Complete()` method exactly once, don't we?

Reply:
that's not part of that commit, is it? (I left this comment on the first line of that commit... I'm not suggesting the `complete` field be dropped)